source/docs/software/advanced-controls/controllers/bang-bang.rst

```diff
@@ -0,0 +1,65 @@
+Bang-Bang Control with BangBangController
+=========================================
+
+.. warning:: Bang-bang control is an extremely aggressive algorithm that relies on response asymmetry to remain stable.  Be *absolutely certain* that your motor controllers have been set to "coast mode" before attempting to control them with a bang-bang controller, or else the braking action will fight the controller and cause potentially destructive oscillation.
```

Comment:
I would move this to the bottom of this section. It will make more sense once BangBang control is explained